network-api/networkapi/buyersguide/templates/about/how_to_use.html

```diff
@@ -13,232 +15,225 @@ <h1 class="h2-heading">
     {% trans "Need help understanding how to use this guide? Here you go:" %}
   </p>
 
-  <section aria-labelledby="minimum-security">
-    <p id="minimum-security"><b>{% trans "Our Minimum Security Standards" %}</b></p>
+  <section aria-labelledby="how-to-use-tip">
+    <p id="how-to-use-tip"><b>{% trans "Tip:" %}</b></p>
     <p class="body">
-      {% blocktrans with url="https://foundation.mozilla.org/privacynotincluded/about/meets-minimum-security-standards" trimmed %}
-      We developed a set of minimum security standards we think all connected products should meet at the <em>very</em>
-      least. Think of it as a “you must be this tall to ride” set of standards. These include five basic things: The
-      product must use encryption, the company must provide automatic security updates, if a product uses a password,
-      it must require a strong password, the company must have a way to manage security vulnerabilities found in their
-      products, and the company must have an accessible privacy policy. If you would like to learn more about our
-      minimum security standards, <a href="{{url}}">go here</a>.
+      {% blocktrans trimmed %}
+      If you don’t find the product you are looking for, search to see if we reviewed any other products made by that company. Often, criteria will match from one product to another from the same company. We simply don’t have time to review all the connected products out there.
       {% endblocktrans %}
     </p>
   </section>
 
-  <section aria-labelledby="encryption">
-    <p id="encryption"><b>{% trans "Encryption" %}</b></p>
+  <section aria-labelledby="warning-label">
+    <p id="warning-label"><b>{% trans "*Privacy Not Included Warning Label" %}</b></p>
```

Comment:
We probably want to use `<strong>` rather than `<b`< if we can, given that we're already using `<em>`.

Reply:
You are correct! I just copied what was there already. I'll update this across the templates – that's an accessibility win.